bin/cli.js

```diff
@@ -473,6 +473,22 @@ For more information, see https://webpack.js.org/api/cli/.`);
 					const statsString = stats.toString(outputOptions);
 					const delimiter = outputOptions.buildDelimiter ? `${outputOptions.buildDelimiter}\n` : "";
 					if (statsString) stdout.write(`${statsString}\n${delimiter}`);
+
+
+					const now = new Date();
```

Comment:
add comment here , maybe abstract to a local function. Needs more comments and explaination.

Reply:
I've added this comment on top:
```js
/**
* Show a hint to donate to our Opencollective
* once a week, only on Monday
*/
```